server/flyway/sql/V26__add_bcsc_user_type_codes.sql

```diff
@@ -0,0 +1,142 @@
+-- Create user type codes to handle 3 types of bcsc users
+
+INSERT INTO app_fam.fam_user_type_code (user_type_code, description) VALUES ('CD', 'User Type for DEV BCSC users.');
+INSERT INTO app_fam.fam_user_type_code (user_type_code, description) VALUES ('CT', 'User Type for TEST BCSC users.');
+INSERT INTO app_fam.fam_user_type_code (user_type_code, description) VALUES ('CP', 'User Type for PROD BCSC users.');
+
+-- Blank out the cognito_user_id field since we are going to have all new users anyway
+
+UPDATE app_fam.fam_user SET cognito_user_id = NULL;
```

Comment:
oh... so we'll have new user pool for prod as well, so the cognito_user_id will be new for all